Git-Workflow-Guidance.md

```diff
@@ -414,62 +479,81 @@ If all you want to do is change the Icepack version in CICE in the repository, t
       git push origin mybranch
       (create PR for CICE)
 
-## Keep Icepack up-to-date when pulling CICE from the Consortium master
-As indicated [above](https://github.com/CICE-Consortium/About-Us/wiki/Git-Workflow-Guidance/#pull-rebase), you need to update the Icepack submodule using 
-```bash
-git submodule update
-```
-after you pull from the Consortium CICE master. This is because Git will fetch submodule changes automatically when doing `git pull` but it will **not** update your local submodule working directory.  
-It might also be necessary to run `git submodule update` after switching branches if the branches record the Icepack submodule at different commits.
+This is the standard procedure for updating the Icepack submodule in CICE.
+
+## Resync Icepack in CICE
+
+As indicated in the [pull and rebase section](https://github.com/CICE-Consortium/About-Us/wiki/Git-Workflow-Guidance/#pull-rebase), you will want to update the Icepack submodule whenever you change branches or pull from another repository or branch.  You can do that with
+
+      git submodule sync
+      git submodule update --init
+
+**NOTE: if you have cloned your own version of Icepack in CICE, do NOT do this as you may lose local changes.**
+
+## Development of Icepack under CICE
 
-## Develop CICE and Icepack concurrently
+When you checkout CICE, a specific version of Icepack will also be cloned.  This version is not ready for modifications, and it can be convenient to develop Icepack changes from a CICE sandbox to facilitate testing within CICE.  You can clone a working version of Icepack to sit within CICE easily as follows,
 
-There are a number of different ways to handle development of Icepack and CICE concurrently.  The easiest way is probably to checkout CICE and Icepack from your fork, and develop each within a single sandbox but as independent branches.  Then do a PR for Icepack first, and finally update Icepack in CICE in your fork and do a CICE PR.  This process looks like
+      git clone https://github.com/username/CICE --recursive CICE.icepack.feature   # checkout CICE
+      cd CICE.icepack.feature
+      mv Icepack Icepack.00                            # move the default Icepack
+      git clone https://github.com/username/Icepack    # clone Icepack separately
```

Comment:
The first block of code seems reasonable.  The second I would not recommend.  I think what I propose is clearest in many ways.  Sure, there is the silly mv and stuff, but what I think it does is reinforce the idea that CICE and Icepack are separate repos.  I also worry in the first approach above that someone will do git push origin feature.  Sure, it won't work if origin in the consortium, but it could cause confusion.  I think clearly checking out a different fork provides some additional clarify for new users.  Again, once folks get familiar, hopefully what you propose is how they start to operate.  But I think we want to tailor this to people who are really inexperienced, and I think the process currently documented is easiest in many ways.